packages/react/src/components/Button/Button.tsx

```diff
@@ -0,0 +1,76 @@
+/**
+ * Copyright (c) 2022, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React, {FC, ReactElement} from 'react';
+import clsx from 'clsx';
+import MuiButton, {LoadingButtonProps as MuiButtonProps} from '@mui/lab/LoadingButton';
+import {WithWrapperProps} from '../../models';
+import {composeComponentDisplayName} from '../../utils';
+
+
```

Comment:
Shall we remove this extra line?

Reply:
Fixed with https://github.com/wso2/oxygen-ui/commit/60d1b964cd88e1d9dc78c533b73150a895b4f2e6